### ISSUE TITLE
lab 방향성 정리

### ISSUE THREAD

**Issue by aisolab** [lassl/scheduler#5]
## 개요
lab 방향성을 새롭게 정리하고자 합니다.

- AsIs
  - 기존에 `hydra`, `pytorch-lightning`, `transformers`, `datasets`, `tokenizers`를 사용하여, pre-trained language model을 쉽게 학습하기위한 pipeline 만들기

- ToBE
  - 기존에 생각한 구현체의 모습이 [lightning-transformers](https://github.com/PyTorchLightning/lightning-transformers)에 거의 담겨있다고 볼 수 있으므로, 실제 구현보다는 위 프로젝트를 최대한 실제로 활용하여, 기 존재하는 pre-trained language model을 실제로 학습을 간단히 해보는 방향으로 진행합니다.
  - 위의 과정에서 알게된 것들을 매 정기모임마다 공유합니다.
  - 공유된 내용을 https://lassl.github.io/ 에 정리해서 공유합니다.

[lightning-transformers](https://github.com/PyTorchLightning/lightning-transformers)

cc: @lassl/authors
